### ISSUE TITLE
Proposal: Disable uri tag for low level http client metrics by default

### ISSUE THREAD

**Issue by luckyswede** [micronaut-projects/micronaut-micrometer#38]
The uri tag in http.client.requests metrics works well for declarative clients since they use the template'd version of the path, however for the low-level client the actual path value is used.
This can cause out-of-memory errors if a lot of different urls are called, since each unique path creates a unique set of tags.

One possibility would be to simply discard the uri tag unless the HttpAttributes.URI_TEMPLATE attribute is set on the http request.

Micronaut 1.2.0
Micrometer-statsd

**Comment by graemerocher** [comment]
This was fixed in 4.8.2 by https://github.com/micronaut-projects/micronaut-micrometer/commit/1b578432f677d667bd96ddb1b4d8c0858391f205

Are you using that version?

**Comment by bjor-joh** [comment]
That was it. It not shows the value of the `uri` tag as `unknown`. Which I guess is expected? If we want to have the path there we need to use the declarative client, right?

Just a side note. I used the micronaut launch to try to reproduce this, but even with the latest version I didn't get this fix included. I had to manually set the version to 4.8.2. 

I realized micronaut-micrometer has not been bumped in the latest releases. Is that intended?
https://github.com/micronaut-projects/micronaut-core/blob/v3.8.9/gradle/libs.versions.toml#L93
